guides/automations.rst

```diff
@@ -386,7 +386,7 @@ All Conditions
 - :ref:`switch.is_on <switch-is_on_condition>` / :ref:`switch.is_off <switch-is_off_condition>`
 - :ref:`sensor.in_range <sensor-in_range_condition>`
 - :ref:`wifi.connected <wifi-connected_condition>` / :ref:`api.connected <api-connected_condition>`
```

Comment:
```suggestion
- :ref:`wifi.connected <wifi-connected_condition>` / :ref:`api.connected <api-connected_condition>`
  / :ref:`mqtt.connected <mqtt-connected_condition>`
```

I'm guessing this is because of a bad merge to current branch? Anyway, please re-add this line.

Reply:
Really sorry about that, fixed 🙂